frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/help-popover/help-popover.component.html

```diff
@@ -0,0 +1,76 @@
+<span
+  [libPopover]="helperTpl"
+  [libPopoverPosition]="popoverPosition"
+  [libPopoverHideDelay]="100"
+  [libPopoverShowDelay]="100"
+  class="truncate"
+>
+  <mat-icon>help</mat-icon>
+</span>
+
+<ng-template #helperTpl>
+  <div class="helper-text">
+    <p>
+      Create a filter query by entering either a property name or just a value
```

Comment:
Should we tag the "i18n" tag on the new text that needs to be translated now? Just tagging the next text, not actually translating it.

Or should we just leave all the tagging + translating to the i18n ticket?